Formula/dotnet.rb

```diff
@@ -67,77 +69,46 @@ class Dotnet < Formula
+    end
 
+    cd "sources" do
       # Use our libunwind rather than the bundled one.
       inreplace "src/runtime/eng/SourceBuild.props",
                 "/p:BuildDebPackage=false",
-                "\\0 --cmakeargs -DCLR_CMAKE_USE_SYSTEM_LIBUNWIND=ON"
-
-      # Fix Clang 15 error: definition of builtin function '__cpuid'.
-      # TODO: Remove with v7.0.0 release which should have merged fix
-      # Ref: https://github.com/dotnet/runtime/commit/992cf8c97cc71d4ca9a0a11e6604a6716ed4cefc
-      inreplace "src/runtime/src/coreclr/vm/amd64/unixstubs.cpp",
-                /^ *void (__cpuid|__cpuidex)\([^}]*}$/,
-                "#if !__has_builtin(\\1)\n\\0\n#endif"
-
-      # Fix missing macOS conditional for system unwind searching.
-      # TODO: Remove with v7.0.0 release which should have merged fix
-      # Ref: https://github.com/dotnet/runtime/commit/97c9a11e3e6ca68adf0c60155fa82ab3aae953a5
-      inreplace "src/runtime/src/native/corehost/apphost/static/CMakeLists.txt",
-                "if(CLR_CMAKE_USE_SYSTEM_LIBUNWIND)",
-                "if(CLR_CMAKE_USE_SYSTEM_LIBUNWIND AND NOT CLR_CMAKE_TARGET_OSX)"
-
-      # Work around arcade build failure with BSD `sed` due to non-compatible `-i`.
-      # TODO: Remove with v7.0.0 release which has removed GNU `sed -i` usage
-      # Ref: https://github.com/dotnet/arcade/commit/b8007eed82adabd50c604a9849277a6e7be5c971
-      inreplace "src/arcade/eng/SourceBuild.props", "\"sed -i ", "\"sed -i.bak " if OS.mac?
-
-      # Workaround for error MSB4018 while building 'installer in tarball' due
-      # to trying to find aspnetcore-runtime-internal v6.0.0 rather than current.
-      # TODO: Remove when packaging is fixed
-      # Issue ref: https://github.com/dotnet/source-build/issues/2795
-      inreplace "src/installer/src/redist/targets/GenerateLayout.targets",
-                "$(MicrosoftAspNetCoreAppRuntimePackageVersion)",
-                "$(MicrosoftAspNetCoreAppRuntimewinx64PackageVersion)"
+                "\\0 /p:EnablePackageValidation=false --cmakeargs -DCLR_CMAKE_USE_SYSTEM_LIBUNWIND=ON"
 
       # Rename patch fails on case-insensitive systems like macOS
       # TODO: Remove whenever patch is no longer used
       rename_patch = "0001-Rename-NuGet.Config-to-NuGet.config-to-account-for-a.patch"
-      (Pathname.pwd/"src/nuget-client/eng/source-build-patches"/rename_patch).unlink if OS.mac?
+      (Pathname("src/nuget-client/eng/source-build-patches")/rename_patch).unlink if OS.mac?
 
-      system "./prep.sh", "--bootstrap"
-      system "./build.sh", "--clean-while-building"
+      system "./prep.sh" # , "--bootstrap"
+      system "./build.sh", "--clean-while-building", "--", "/v:n",
+                                                           "/p:LogVerbosity=n",
+                                                           "/p:MinimalConsoleLogOutput=false",
+                                                           "/p:ContinueOnPrebuiltBaselineError=true"
```

Comment:
Will note that I am not sure on how `--bootstrap` flag should be used. Upstream mentions mainly needed on arm64, but I think it was also needed on Intel macOS (though source-build isn't officially supported on macOS). May not be needed on Intel Linux.

The current stalling is happening in `./build.sh` step.